receiver/azureeventhubreceiver/azureresourcelogs_unmarshaler_test.go

```diff
@@ -236,7 +236,7 @@ func TestUnmarshalLogs(t *testing.T) {
 	expectedMinimum := plog.NewLogs()
 	resourceLogs := expectedMinimum.ResourceLogs().AppendEmpty()
 	scopeLogs := resourceLogs.ScopeLogs().AppendEmpty()
-	scopeLogs.Scope().SetName("otelcol/" + typeStr)
```

Comment:
The scope name is changed here, this should be called out with a changelog entry

Reply:
ok then, let's slow down and talk about it as part of #21382